### PR TITLE
Development

### PR DESCRIPTION
**Pull request by JLee2011**
there should be a conflict in the readme file before merging dev to main...resolve it please.

### CODE REVIEW

File: marcio.js [hunk 1]
```diff
@@ -0,0 +1 @@
+console.log("why am i not printing in the terminal.  fix me thanks.")
```
Comment (Omarci55) [inline marcio.js:1]: All good !

**Review by Omarci55 — COMMENTED** [review]
Some comments left on each file

**Review by lllhollliday — COMMENTED** [review]
hmm it doesn't seem to be flagged as a conflict.. can you try again?

**Review by Omarci55 — APPROVED** [review]

**Comment by Omarci55** [comment]
There's no conflict falged top me. I have only marcio.js with one change

**Pull request merged**